test/unit/core/Raycaster.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @author simonThiele / https://github.com/simonThiele
+ */
+
+module( "Raycaster" );
+
+test( "intersectObjects", function() {
+	var raycaster = getRaycaster();
+	var objectsToCheck = getObjectsToCheck();
+
+	ok ( raycaster.intersectObjects( objectsToCheck ).length === 1,
+		"no recursive search should lead to one hit" );
+
+	ok ( raycaster.intersectObjects( objectsToCheck, true ).length === 3,
+		"recursive search should lead to two hits" );
```

Comment:
Sry, I wrote the test with two sphere first and then forgot to change the comment. Thx!